bundles/org.openhab.binding.snmp/src/main/resources/OH-INF/i18n/snmp.properties

```diff
@@ -85,8 +85,8 @@ channel-type.config.snmp.number.mode.option.READ_WRITE = Read/Write
 channel-type.config.snmp.number.mode.option.TRAP = Trap
 channel-type.config.snmp.number.oid.label = OID
 channel-type.config.snmp.number.oid.description = OID in dotted format (eg. .1.3.6.1.4.1.6574.3.1.1.3.0)
-channel-type.config.snmp.number.unit.label = Unit Of Measurement
-channel-type.config.snmp.number.unit.description = Unit of measurement (optional). The unit is used for representing the value in the GUI as well as for converting incoming values (like from '°F' to '°C'). Examples: "°C", "°F"
+channel-type.config.snmp.number.unit.label = Unit
```

Comment:
I think the current value is better, could you alter the xml? Also for description

Reply:
Done.